connector/aws/ssm/ssm.go

```diff
@@ -0,0 +1,67 @@
+package ssm
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"os"
+)
+
+var AwsSsmRegion = os.Getenv("AWS_REGION")
```

Comment:
AwsSsmRegion  -->  AWS_SSM_REGION,  keep the variable name in the same format, some are in camel case and some in snake case.